pvlib/iotools/psm3.py

```diff
@@ -117,6 +117,11 @@ def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
        <https://developer.nrel.gov/docs/solar/nsrdb/psm3-tmy-download/>`_
     .. [4] `Physical Solar Model (PSM) v3 - Five Minute Temporal Resolution
        <https://developer.nrel.gov/docs/solar/nsrdb/psm3-5min-download/>`_
+
+    .. versionchanged:: 0.9.0
+       The function now returns a tuple where the first element is a dataframe
+       and the second element is a dictionary containing metadata. Previous
+       versions of this function had the return values switched.
```

Comment:
Maybe put these `versionchanged` entries above the `Parameters` sections so that people might actually see them?  I think it will render correctly up there...

Reply:
You are indeed correct. I had tried other places but not at the very top.